roles/mdc/tasks/install-console.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Process mdc template
+  template:
+    src: mobiledeveloperconsole.yml.j2
+    dest: "/tmp/mobiledeveloperconsole.yml"
+
+- name: Apply mdc resource
+  shell: "oc apply -f /tmp/mobiledeveloperconsole.yml -n {{ mobile_services_namespace }}"
+
+- name: Delete mdc template file
+  file:
+    path: /tmp/mobiledeveloperconsole.yml
+    state: absent
+
+- name: Wait for mdc readiness
+  shell: "oc get mobiledeveloperconsole/{{ mdc_name }} -o jsonpath='{.status.phase}' -n {{ mobile_services_namespace }}"
```

Comment:
This check seems to wait more thoroughly than checking that the DeploymentConfig status is ready which is done for other checks.

Reply:
Although when it reached the end of the 50 retries it continued even though the pod had not come up due to memory issues. The playbook failed at a later task.